decentraland/unity-renderer
# feat: teleport modifiers for worlds

## What does this PR change?

This has no real change in renderer, adding a crash log just to create the PR, so we can test that teleport is working correctly on the respective kernel branch. Once this has been tested, this branch will be closed. 

## How to test the changes?

Open DCL using https://play.decentraland.zone/?renderer-branch=feat/teleport-modifiers-for-worlds&kernel-branch=feat/teleport-worlds-commands

Check that the following teleport commands work correctly as it currently works:

1) /goto home
2) /goto 0,0 (or any coordinate)
3) /goto random
4) Teleport from map
5) Teleport from Highlights tab
6) Teleport from Places tab
7) Teleport from Events tab

After that, repeat the test, but starting each step from the `menduz.dcl.eth` realm. Meaning, teleport to that realm using `/changerealm menduz.dcl.eth` and test that every teleport command teleports you correctly to the place it should go.


## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
